cli/cmd/report_rules.go

```diff
@@ -0,0 +1,399 @@
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package cmd
+
+import (
+	"fmt"
+	"sort"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/fatih/structs"
+	"github.com/olekukonko/tablewriter"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/lacework/go-sdk/api"
+)
+
+var (
+	// report-rules command is used to manage lacework report rules
+	reportRulesCommand = &cobra.Command{
+		Use:     "report-rule",
+		Aliases: []string{"report-rules", "rr"},
+		Short:   "Manage report rules",
+		Long: `Manage report rules to route reports to one or more email alert channels.		
+
+An report rule has Four parts:
+
+  1. Email Alert channel(s) that should receive the report
+  2. Severities to include
+  3. Resource group(s) containing the subset of your environment to consider
+  4. Notification types containing which report information to send
+`,
+	}
+
+	// list command is used to list all lacework report rules
+	reportRulesListCommand = &cobra.Command{
+		Use:     "list",
+		Aliases: []string{"ls"},
+		Short:   "List all report rules",
+		Long:    "List all report rules configured in your Lacework account.",
+		Args:    cobra.NoArgs,
+		RunE: func(_ *cobra.Command, _ []string) error {
+			reportRules, err := cli.LwApi.V2.ReportRules.List()
+			if err != nil {
+				return errors.Wrap(err, "unable to get report rules")
+			}
+			if len(reportRules.Data) == 0 {
+				msg := `There are no report rules configured in your account.
+
+Get started by integrating your report rules to manage reporting using the command:
+
+    lacework report-rule create
+
+If you prefer to configure report rules via the WebUI, log in to your account at:
+
+    https://%s.lacework.net
+
+Then navigate to Settings > Report Rules.
+`
+				cli.OutputHuman(fmt.Sprintf(msg, cli.Account))
+				return nil
+			}
+			if cli.JSONOutput() {
+				return cli.OutputJSON(reportRules)
+			}
+
+			var rows [][]string
+			for _, rule := range reportRules.Data {
+				rows = append(rows, []string{rule.Guid, rule.Filter.Name, rule.Filter.Status()})
+			}
+
+			cli.OutputHuman(renderSimpleTable([]string{"GUID", "NAME", "ENABLED"}, rows))
+			return nil
+		},
+	}
+	// show command is used to retrieve a lacework report rule by resource id
+	reportRulesShowCommand = &cobra.Command{
+		Use:   "show",
+		Short: "Show an report rule by id",
+		Long:  "Show a single report rule by it's ID.",
```

Comment:
either `ID` or `id` but be consistent on all other commands
```suggestion
		Long:  "Show a single report rule by it's id.",
```